src/renderer/components/ClusterSettings/Preferences/index.vue

```diff
@@ -70,18 +80,34 @@ export default {
       errors: {
         terminalcwd: null
       },
-      prometheusPath: ""
+      prometheusPath: "",
+      prometheusProvider: "",
+      prometheusProviders: [
```

Comment:
Probably should use `PrometheusProviderRegistry` ?

Reply:
both `main` and `renderer` shares now the same registry implementation.